charts/gitprime-app/v3.0.5/templates/dataPipeline/_deployment.tpl

```diff
@@ -0,0 +1,188 @@
+              value: {{ quote .templateData.newThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.newMaxCommitCount }}
+            - name: GP_NEW_MAX_COMMIT_COUNT
+              value: {{ quote .templateData.newMaxCommitCount }}
+          {{- end}}
+          {{- if .templateData.incrListeners }}
+            - name: GP_INCREMENTAL_LISTENERS
+              value: {{ quote .templateData.incrListeners }}
+          {{- end}}
+          {{- if .templateData.incrThreadsPerListener }}
+            - name: GP_INCREMENTAL_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.incrThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.incrMaxCommitCount }}
+            - name: GP_INCREMENTAL_MAX_COMMIT_COUNT
+              value: {{ quote .templateData.incrMaxCommitCount }}
+          {{- end}}
+          {{- if .templateData.aodListeners }}
+            - name: GP_AOD_LISTENERS
+              value: {{ quote .templateData.aodListeners }}
+          {{- end}}
+          {{- if .templateData.aodThreadsPerListener }}
+            - name: GP_AOD_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.aodThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.ticketListeners }}
+            - name: GP_TICKET_LISTENERS
+              value: {{ quote .templateData.ticketListeners }}
+          {{- end}}
+          {{- if .templateData.ticketThreadsPerListener }}
+            - name: GP_TICKET_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.ticketThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.prListeners }}
+            - name: GP_PR_LISTENERS
+              value: {{ quote .templateData.prListeners }}
+          {{- end}}
+          {{- if .templateData.prThreadsPerListener }}
+            - name: GP_PR_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.prThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.prWebhookListeners }}
+            - name: GP_PR_WEBHOOK_LISTENERS
+              value: {{ quote .templateData.prWebhookListeners }}
+          {{- end}}
+          {{- if .templateData.prWebhookThreadsPerListener }}
+            - name: GP_PR_WEBHOOK_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.prWebhookThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.ticketWebhookListeners }}
+            - name: GP_TICKET_WEBHOOK_LISTENERS
+              value: {{ quote .templateData.ticketWebhookListeners }}
+          {{- end}}
+          {{- if .templateData.ticketWebhookThreadsPerListener }}
+            - name: GP_TICKET_WEBHOOK_WORKERS_PER_LISTENER
+              value: {{ quote .templateData.ticketWebhookThreadsPerListener }}
+          {{- end}}
+          {{- if .templateData.repoDeleteListeners }}
+            - name: GP_REPO_DELETE_LISTENERS
+          value: {{ quote .templateData.repoDeleteListeners }}
+          {{- end}}
```

Comment:
Not sure if indention matters here with the yaml.

Reply:
Great catch! 👍